test/controllers/communities_controller_test.rb

```diff
@@ -5,17 +5,23 @@ class CommunitiesControllerTest < ActionDispatch::IntegrationTest
     @community = communities(:one)
   end
 
-  test "should get index" do
+  test 'should get index' do
     get communities_url
     assert_response :success
   end
 
-  test "should get new" do
+  #test 'should redirect to show if there are only one community' do
```

Comment:
Yes, in general I think we should be avoiding committing commented code to master.  I want to try and keep this project as clean as possible.  Hence the stuff about getting rid of the empty css, coffeescript files, etc....

If this is stuff that should be kept for some reason, like a later feature then it can be put into it's own branch.